mitmproxy/tools/web/app.py

```diff
@@ -237,6 +237,8 @@ def wrapper(
         ) -> R | None:
             if not self.current_user:
                 password = self.get_argument("token", default="")
+                if not password:
+                    password = self.request.headers.get("X-Token") or ""
```

Comment:
Is there a reason you went with this over the more standardized `Authorization: Bearer ...`?

Reply:
That was a quick way to do it, but `Authorization: Bearer ...` is way cleaner. Updated in f02eb9d48a326853b56c7a31c605de45a660ca5c